lib/grape-swagger/doc_methods/parse_params.rb

```diff
@@ -64,7 +64,9 @@ def document_array_param(value_type)
           if value_type[:is_array]
             if value_type[:documentation].present?
               param_type = value_type[:documentation][:param_type]
-              type = GrapeSwagger::DocMethods::DataType.mapping(value_type[:documentation][:type])
+              if (doc_type = value_type[:documentation][:type])
```

Comment:
Rubocop should have complained, that `if` should be on the right hand side of the assignment.